libstorage/drivers/storage/rbd/utils/utils.go

```diff
@@ -177,6 +183,28 @@ func GetMappedRBDs(ctx types.Context) (map[string]string, error) {
 	return devMap, nil
 }
 
+//parseMappedRBDs is a two-stage parse attempt because Ceph v13 changed JSON format
+func parseMappedRBDs(output []byte) ([]*rbdMappedEntry, error) {
+	rbdMap := []*rbdMappedEntry{}
+	rbdMapPreMimic := map[string]*preMimicRbdMappedEntry{}
+
+	err := json.Unmarshal(output, &rbdMapPreMimic)
+	if err != nil {
+
+		err2 := json.Unmarshal(output, &rbdMap)
+		if err2 != nil {
+			return nil, goof.WithError(
+				"unable to parse rbd showmapped", err2)
+		}
+
+	} else {
+		for index, entry := range rbdMapPreMimic {
+			rbdMap = append(rbdMap, &rbdMappedEntry{ID: index, Device: entry.Device, Name: entry.Name, Pool: entry.Pool, Snap: entry.Snap})
```

Comment:
Since you are going to need to squash and rebase anyways, I'd like to request a stylistic change. Please try to adhere to the 80 character max style, and put this line into multiple lines, akin to [this example](https://github.com/rexray/rexray/blob/6340189e4ae317293cb005eb7e30230380250d27/libstorage/drivers/storage/rbd/storage/rbd_storage.go#L143-L147).

meaning, like this:
```go
		        rbdMap = append(rbdMap, &rbdMappedEntry{
                             ID: index,
                             Device: entry.Device,
                             Name: entry.Name,
                             Pool: entry.Pool,
                             Snap: entry.Snap})
```